WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <%= render layout: "nav" do %>
   <% cache ["registrations_index", @registrations.pluck(:id), @registrations.maximum(:updated_at)] do %>
-    <%= wca_table data: { toggle: "table", sort_name: "name" } do %>
```

Comment:
This should probably be left in, since it shows the little blue arrow indicating we're sorted on the Name column.

Or so I assume.